src/components/map/MapDirective.js

```diff
@@ -80,6 +81,40 @@ goog.require('ga_permalink');
 
             map.setTarget(element[0]);
 
+            scope.$watch('::ol3d', function(ol3d) {
+              if (ol3d) {
+                var camera = ol3d.getCesiumScene().camera;
+                var params = gaPermalink.getParams();
+
+                // initial location based on query params
+                var position = undefined;
+                if (params.lon && params.lat && params.elevation) {
+                  var lon = parseFloat(params.lon);
+                  var lat = parseFloat(params.lat);
+                  var elevation = parseFloat(params.elevation);
+                  position = new Cesium.Cartographic(lon, lat, elevation);
+                }
+                camera.setView({
+                  positionCartographic: position,
+                  heading: params.heading ? parseFloat(params.heading) : undefined,
+                  pitch: params.pitch ? parseFloat(params.pitch) : undefined,
+                  roll: 0.0
+                });
+
+                // update permalink
+                camera.moveEnd.addEventListener(function() {
```

Comment:
I don't know how often this even is triggered. But we should probably debounce this update with gaDebounce (as is done with map parameters: https://github.com/geoadmin/mf-geoadmin3/blob/d88ca65a4b5592f375625c7264c597f6b070ee2b/src/components/map/MapDirective.js#L77)